client/my-sites/google-my-business/stats/chart.js

```diff
@@ -6,19 +6,21 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { isEqual, get } from 'lodash';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import Card from 'components/card';
 import CardHeading from 'components/card-heading';
-import FakeData from './fake-data';
 import PieChart from 'components/pie-chart';
 import PieChartLegend from 'components/pie-chart/legend';
 import SectionHeader from 'components/section-header';
-import { changeGoogleMyBusinessStatsInterval } from 'state/google-my-business/actions';
-import { getInterval } from 'state/google-my-business/selector';
+import {
+	changeGoogleMyBusinessStatsInterval,
+	requestGoogleMyBusinessStats,
```

Comment:
I think the common pattern is to use the `fetch` term when retrieving data. Shouldn't we rename that action into `fetchGoogleMyBusinessStats`?

Reply:
Looks like `request` is perfectly fine.

Reply:
I've seen `request` used elsewhere, so I went with it.